apps/archive/archive_spike.py

```diff
@@ -162,6 +164,30 @@ def update(self, id, updates, original):
             updates['previous_marked_user'] = original['marked_for_user']
             updates['marked_for_user'] = None
 
+        if original.get('translation_id') and original.get('translated_from'):
+            # remove translations info from the translated item on spike
+            updates['translated_from'] = None
+            updates['translation_id'] = None
+
+            id_to_remove = original.get('_id')
```

Comment:
I might be wrong, but I think it is `guid`.

Reply:
_id field goes to `translations`, so it to remove should be _id as well